SRC/dlasrti.f

```diff
@@ -0,0 +1,315 @@
+*> \brief \b DLASRTI sorts array indices based on the referenced numbers
+*
+*  =========== DOCUMENTATION ===========
+*
+* Online html documentation available at
+*            http://www.netlib.org/lapack/explore-html/
+*
+*> \htmlonly
+*> Download DLASRTI + dependencies
+*> <a href="http://www.netlib.org/cgi-bin/netlibfiles.tgz?format=tgz&filename=/lapack/lapack_routine/dlasrti.f">
+*> [TGZ]</a>
+*> <a href="http://www.netlib.org/cgi-bin/netlibfiles.zip?format=zip&filename=/lapack/lapack_routine/dlasrti.f">
+*> [ZIP]</a>
+*> <a href="http://www.netlib.org/cgi-bin/netlibfiles.txt?format=txt&filename=/lapack/lapack_routine/dlasrti.f">
+*> [TXT]</a>
+*> \endhtmlonly
+*
+*  Definition:
+*  ===========
+*
+*       SUBROUTINE DLASRTI( ID, N, X, INDICES, INFO )
+*
+*       .. Scalar Arguments ..
+*       CHARACTER          ID
+*       INTEGER            INFO, N
+*       ..
+*       .. Array Arguments ..
+*       DOUBLE PRECISION   X( * )
+*       INTEGER            INDICES( * )
+*       ..
+*
+*
+*> \par Purpose:
+*  =============
+*>
+*> \verbatim
+*>
+*> Sort the numbers in X indirectly in increasing order (if ID = 'I') or
+*> in decreasing order (if ID = 'D' ) using the array of INDICES.
+*>
+*> Use Quick Sort, reverting to Insertion sort on arrays of
+*> size <= 20. Dimension of STACK limits N to about 2**32.
+*> \endverbatim
+*
+*  Arguments:
+*  ==========
+*
+*> \param[in] ID
+*> \verbatim
+*>          ID is CHARACTER*1
+*>          = 'I': sort X in increasing order;
+*>          = 'D': sort X in decreasing order.
+*> \endverbatim
+*>
+*> \param[in] N
+*> \verbatim
+*>          N is INTEGER
+*>          The length of the array X and INDICES.
+*> \endverbatim
+*>
+*> \param[in] X
+*> \verbatim
+*>          X is DOUBLE PRECISION array, dimension (N)
+*>          The values to be sorted.
+*> \endverbatim
+*>
+*> \param[in,out] INDICES
+*> \verbatim
+*>          X is INTEGER array, dimension (N)
+*>          On entry, the indices of values in X to be sorted.
+*>          On exit, X has been sorted into
+*>          increasing order such that
+*>          X( INDICES(1) ) <= ... <= X( INDICES(N) )
+*>          or decreasing order such that
+*>          X( INDICES(1) ) >= ... >= X( INDICES(N) )
+*>          depending on ID.
+*> \endverbatim
+*>
+*> \param[out] INFO
+*> \verbatim
+*>          INFO is INTEGER
+*>          = 0:  successful exit
+*>          < 0:  if INFO = -i, the i-th argument had an illegal value
+*> \endverbatim
+*
+*  Authors:
+*  ========
+*
+*> \author Christoph Conrads (https://christoph-conrads.name)
+*> \author Univ. of Tennessee
+*> \author Univ. of California Berkeley
+*> \author Univ. of Colorado Denver
+*> \author NAG Ltd.
+*
+*> \date April 2020
+*
+*> \ingroup auxOTHERcomputational
+*
+*  =====================================================================
+      SUBROUTINE DLASRTI( ID, N, X, INDICES, INFO )
+*
+*  -- LAPACK computational routine (version TODO) --
```

Comment:
Remove TODO before merging

Reply:
Please confirm that the next version is 3.10.0.

Reply:
At least that is what langou told me some weeks ago

Reply:
commit c3c119c104fc2ec2179386d17726663bb37cb061

Reply:
Yes, yes, next release is 3.10.0. I see that @christoph-conrads edited with 3.10.0, great. And February 2021 as release date. February 2021 as release date will be a good challenge. I think we can make it. 15 days. A lots of code review. We have the out-of-bound problem in TSQR that needs to be looked into, and then I would have liked to release @christoph-conrads with on GSVD ( #406 ) and @thijssteel work on AED for QZ ( #421 ). @weslleyspereira and I will start reviewing these codes.